livre/core/Render/GLContext.h

```diff
@@ -43,6 +43,8 @@ class GLContext : public boost::enable_shared_from_this< GLContext >
      */
     void shareContext( GLContextPtr dstContextPtr );
 
+    const GLEWContext* glewGetContext() const;
```

Comment:
Not sure I understand the question. Either the 3rd-party uses GLEW, then it will accept a GLEWContext, or not, in which case it will have to do it's own wglGetProcAddress magic for Windows. There is no hidden side effect of using GLEW, other then that it redefines all glFoo to glewGetContext()->__glewFoo